utils/data/fuse/bad_fuse_targs.yaml

```diff
@@ -1,22 +1,10 @@
 # For completeness, this is a list of all ULLYSES targets that
 # have FUSE data, but data were not used due to data quality issues.
 # It may be possible to address these issues in the future.
+# This list is not used in any way, just kept for informational purposes.
 - AV83
```

Comment:
I think that AV83 should also be removed: https://jira.stsci.edu/browse/ULLYSES-1257

Reply:
Note- its populated correctly elsewhere.